application/comit_node/tests/htlc_harness/mod.rs

```diff
@@ -19,3 +22,26 @@ pub fn new_account(secret_key: &str) -> (KeyPair, Address) {
 
 pub const SECRET: &[u8; 32] = b"hello world, you are beautiful!!";
 pub const HTLC_TIMEOUT: Duration = Duration::from_secs(5);
+
+#[derive(Debug)]
+pub struct CustomSizeSecret(pub Vec<u8>);
+
+impl CustomSizeSecret {
+    pub fn hash(&self) -> SecretHash {
+        let mut sha = Sha256::new();
+        sha.input(&self.0[..]);
+
+        let mut result: [u8; SecretHash::LENGTH] = [0; SecretHash::LENGTH];
+        sha.result(&mut result);
+        SecretHash::from(result)
+    }
+}
+
+impl FromStr for CustomSizeSecret {
+    type Err = FromHexError;
+
+    fn from_str(s: &str) -> Result<Self, <Self as FromStr>::Err> {
```

Comment:
❓ does this work like that?

Reply:
Yes, it's the same than what we are doing in `htlc_harness/mod.rs`:
`pub const SECRET: &[u8; 32] = b"hello world, you are beautiful!!";`